optimization/@matRad_OptimizationProblemDAO/matRad_getJacobianStructure.m

```diff
@@ -73,7 +73,7 @@
     end
 
     % second leaves
-    counter = 0;
+    counter = i;
```

Comment:
I would say

counter = apertureInfo.totalNumOfShapes

to be more explicit here.

Reply:
yeah that's right